tests/rest_api_tests.go

```diff
@@ -1,16 +1,21 @@
 package main
 
 import (
-	"github.com/redhatinsighs/insights-operator-controller/tests/rest"
+	"github.com/redhatinsighs/insights-operator-controller/server"
+	tests "github.com/redhatinsighs/insights-operator-controller/tests/rest"
 	"github.com/verdverm/frisby"
 )
 
 func main() {
-	tests.ServerTests()
-	tests.ClusterTests()
-	tests.ConfigurationTests()
-	tests.OperatorTests()
-	tests.ProfileTests()
-	tests.TriggerTests()
+	if server.Environment != "production" {
+		tests.ServerTests()
+		tests.ClusterTests()
+		tests.ConfigurationTests()
+		tests.OperatorTests()
+		tests.ProfileTests()
+		tests.TriggerTests()
+	} else {
+		tests.AuthTests()
+	}
```

Comment:
Maybe we should require a LDAP token (and mention it in README) and run all the tests regardless the environment.